pkg/utils/kv_auth.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/gtank/cryptopasta"
+	"go.etcd.io/etcd/clientv3"
+)
+
+const etcdKvModeAuthKeyPath = "/dashboard/kv_mode/auth_key"
```

Comment:
Maybe better to call it `kv_auth` directly since there will be no other entities in `kv_mode`.